tc/core/cpu/cpu_rtc.h

```diff
@@ -25,5 +26,9 @@ struct CpuRTCFunction {
   void clear() {}
   template <typename... Args>
   void operator()(Args&... args) {}
+  const std::string& deviceName() const {
+    static std::string dummy;
+    return dummy;
+  }
```

Comment:
Shouldn't this hunk go into c31ae2ba1d54720f3c33f2617e54493ebfce42d1,
or at least before 14e02cf693ac81bbbe8fe7c598412638e6838884?